### PR TITLE
Merge the webpack and first setup

### PR DESCRIPTION
**Pull request by ArcaDelDelirio**
We need to merge the new setup to master and deployment the page to gh-page

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "presets": [
```
Comment (EricDS-INFO) [inline .babelrc]: What the f**k is this ???
Sorry for not understanding WebPack :,D

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "all",
+    "tabWidth": 2,
+    "semi": false,
```
Comment (EricDS-INFO) [inline .prettierrc]: Pablo and I suggest to keep the semicolons in order to understand the code we write. We are probbably getting lost without them

Comment (ArcaDelDelirio) [reply .prettierrc]: Okei i got it !!

File: .babelrc (hunk 1 above)
Comment (ArcaDelDelirio) [reply .babelrc]: This is the official documentation https://babeljs.io/docs/en/babel-preset-env , but basically the preset are a plugin develop by third party to allow certain functionality to babel core, like and ad-hoc approach

**Pull request merged**